### ISSUE TITLE
King_DuckZ items

### ISSUE THREAD

**Issue by necrose99** [necrose99/necromancy-overlay#7]
@KingDuckZ  afew of king_duckz items might be worth a whack in attempts at packaging 
ie tigersum etc cmake relate ebuilds are easy to covert to  app-crypt/tigersum

https://wiki.gentoo.org/wiki/Dlang
https://bitbucket.org/King_DuckZ/

**Comment by KingDuckZ** [comment]
Just a heads up on this if you guys are still interested: most of my stuff has moved to https://alarmpi.no-ip.org/gitan/King_DuckZ and what has not moved yet will eventually follow. In particular Tigersum should now be easier to compile due to gdc becoming part of official gcc now, and me using Meson as the build system (instead of the old patchy cmake approach).

**Comment by necrose99** [comment]
yeah these are mostly testing or victim e builds from time to time.  but will take a look when i get a few...

**Comment by KingDuckZ** [comment]
Cool :) Also, I forgot to mention, some of my libs are already packaged in [gentoo guru](https://github.com/gentoo/guru), so you might want to take a look there too!

**Comment by necrose99** [comment]
@KingDuckZ  @pentoo also for more the crypto sec items not pkg'd
